common/country_types/hfe_country_types.txt

```diff
@@ -209,13 +249,25 @@ hfe_drone = {
 					base = 33
 				}
 			}
+			ancient_drone_station = {
```

Comment:
What is this and why it here

Reply:
The Ancient Drone Stations are part of the Drone Core system for both the Harvesting Marvel and the vanilla rendition Ancient Mining Drones. They were instituted by Paradox and are part of the things you must kill to access Ancient Mining Drone systems.

Reply:
How can they build it?

Reply:
Ask Paradox.

Reply:
Nohow, they cant have acces to this shipsize.

Reply:
Whatever.

Reply:
Remove it

Reply:
Okay, done